src/constant/index.js

```diff
@@ -291,3 +291,41 @@ export const CROSS_SELL_PRODUCT_IDS_MAP = IS_TESTNET
         'likenft1r8mce5h96nvedtm7rh9va5mda6fgysqfprlxftuef09s8kfjt2jsqeyyjq',
       ],
     };
+
+export const AFFILIATION_CHANNELS = [
+  { id: 'liker_land', name: { en: 'Liker Land', 'zh-Hant': 'Liker Land' } },
+  { id: '@bookpunch', name: { en: 'bookpunch', 'zh-Hant': '一拳書店' } },
+  {
+    id: '@boundarybooks',
+    name: { en: 'boundarybooks', 'zh-Hant': '界限書店' },
+  },
+  {
+    id: '@breakthrough_publish',
+    name: { en: 'breakthrough_publish', 'zh-Hant': '突破書廊' },
+  },
+  {
+    id: '@hansbookstore',
+    name: { en: 'hansbookstore', 'zh-Hant': '留下書店' },
+  },
+  {
+    id: '@hkhouseofliterature',
+    name: { en: 'hkhouseofliterature', 'zh-Hant': '香港文學生活館' },
+  },
+  {
+    id: '@hkreaders',
+    name: { en: 'hkreaders', 'zh-Hant': '序言書室' },
+  },
+  {
+    id: '@hunterbookstore',
+    name: { en: 'hunterbookstore', 'zh-Hant': '獵人書店' },
+  },
+  { id: '@kubrick_hk', name: { en: 'kubrick_hk', 'zh-Hant': 'Kubrick' } },
+  {
+    id: '@nowherebooks',
+    name: { en: 'nowherebooks', 'zh-Hant': '飛地書店' },
+  },
+  {
+    id: '@samkeebook',
+    name: { en: 'samkeebook', 'zh-Hant': '森記圖書公司' },
+  },
+];
```

Comment:
```suggestion
export const AFFILIATION_CHANNELS = [
  { id: 'liker_land', name: { en: 'Liker Land', 'zh-Hant': 'Liker Land' } },
  { id: '@bookpunch', name: { en: 'Book Punch', 'zh-Hant': '一拳書店' } },
  {
    id: '@boundarybooks',
    name: { en: 'Boundary Bookstore', 'zh-Hant': '界限書店' },
  },
  {
    id: '@breakthrough_publish',
    name: { en: 'Breakthrough Book Gallery', 'zh-Hant': '突破書廊' },
  },
  {
    id: '@hansbookstore',
    name: { en: 'Have A Nice Stay', 'zh-Hant': '留下書舍' },
  },
  {
    id: '@hkhouseofliterature',
    name: { en: 'House of HKLIT', 'zh-Hant': '香港文學生活館' },
  },
  {
    id: '@hkreaders',
    name: { en: 'Hong Kong Reader', 'zh-Hant': '序言書室' },
  },
  {
    id: '@hunterbookstore',
    name: { en: 'Hunter Bookstore', 'zh-Hant': '獵人書店' },
  },
  { id: '@kubrick_hk', name: { en: 'kubrick_hk', 'zh-Hant': 'Kubrick' } },
  {
    id: '@nowherebooks',
    name: { en: 'Nowhere Bookstore', 'zh-Hant': '飛地書店' },
  },
  {
    id: '@samkeebook',
    name: { en: 'Samkee Book', 'zh-Hant': '森記圖書公司' },
  },
];
```

Reply:
[💬 Update names in affiliation channel list](https://github.com/likecoin/liker-land/pull/1751/commits/6cd4b62f639045c85bfc1f44c032d139d7ce4122)